platform/broadcom/sai.mk

```diff
@@ -1,6 +1,6 @@
-LIBSAIBCM_XGS_VERSION = 12.3.2.2
+LIBSAIBCM_XGS_VERSION = 12.0.0.8
 LIBSAIBCM_DNX_VERSION = 12.3.2.2
-LIBSAIBCM_XGS_BRANCH_NAME = SAI_12.3.0_master
+LIBSAIBCM_XGS_BRANCH_NAME = SAI_12.0.0_EA3
```

Comment:
@developfast - should this be SAI_12.5.0_EA?

Reply:
Yes @tjchadaga this will be moved to SAI_12.5.0_EA.

12.0.0EA3 is the pre-release version that we got from the vendor.